tools/deployment/package-helm/templates/service-account-rolebinding.yaml

```
@@ -1,14 +1,14 @@
 apiVersion: "rbac.authorization.k8s.io/v1"
 kind: "RoleBinding"
 metadata:
-  name: {{ include "clp.fullname" . }}-job-watcher
+  name: {{ include "clp.fullname" . }}-service-account
```

YAMLlint (1.38.0)

[error] too many spaces inside braces

(braces)